src/cryptenroll/cryptenroll.c

```diff
@@ -103,6 +103,50 @@ static const char *const luks2_token_type_table[_ENROLL_TYPE_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(luks2_token_type, EnrollType);
 
+static int determine_default_node(void) {
+        int r;
+
+        dev_t devno;
+        r = get_block_device("/var", &devno);
```

Comment:
I think the rationale of choosing `/var/` should really be described in a comment too